### PR TITLE
Release v2.2.0

### PR DESCRIPTION
**Pull request by uvvpavel**

### CODE REVIEW

**Review by lucianomartin — APPROVED** [review]
No comments from my side.

File: test/filter_tests/script/test_filter_biquad_sat_s32_case3.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+# print(f"N_samples: {N_samples}")
+
+for i in range(len(samples)):
+    y[i] = ts.biquads(samples[i], N_biquads, state, coef)
+    y_q30[i] = biquads_q30_sat(samples[i], N_biquads, state_q30, coef_q30)
+
+diff = y - y_q30
+
+
+# print("y[] = ", y)
```
Comment (mbanth) [inline test/filter_tests/script/test_filter_biquad_sat_s32_case3.py:88]: Why has commented out code been left in?

**Review by mbanth — APPROVED** [review]
I left one question.  However, it certainly is not important enough to hold up the release.  Otherwise, everything looks good.

File: test/filter_tests/script/test_filter_biquad_sat_s32_case3.py (hunk 1 above)
Comment (uvvpavel) [reply test/filter_tests/script/test_filter_biquad_sat_s32_case3.py:88]: I think Allan used it for debugging and never removed it properly. I think it's fine for the test script

**Pull request merged**